### PR TITLE
About-us component

### PR DESCRIPTION
**Pull request by arc9693**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,10 +3,15 @@
...
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.2/css/bootstrap.min.css" integrity="sha384-Smlep5jCw/wG7hdkwQ/Z5nLIefveQRIY9nfy6xoR1uRYBtpZgI6339F5dgvm/e9B" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.2.0/css/all.css" integrity="sha384-hWVjflwFxL6sNzntih27bfxkr27PmbbK/iSvJ+a4+0owXq79v+lsFkW54bOGbiDQ" crossorigin="anonymous">
     <title>lugnitdgp-website</title>
   </head>
   <body>
     <div id="app"></div>
     <!-- built files will be auto injected -->
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+   <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js" integrity="sha384-ZMP7rVo3mIykV+2+9J3UJ46jBk0WLaUAdn689aCwoqbBJiSnjAK/l8WvCWPIPm49" crossorigin="anonymous"></script>
```
Comment (ankan17) [inline index.html]: Be consistent with the indentation

File: src/components/Home.vue [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
+    <about-us/>
   </div>
 </template>
 
 <script>
+import aboutUs from './About-us'
 export default {
-  name: 'Home'
+  name: 'Home',
+  components: {aboutUs}
```
Comment (ankan17) [inline src/components/Home.vue]: Make your components starting with capital letter. This is the convention. e.g., AboutUs, AnimateNumber, etc.

File: src/components/animateNumber.vue [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+}
+</script>
+
+<style scoped>
+.highlighted{
+  color:#fa631c !important;
+}
+h3{
+  color:white;
+  margin: 0 !important;
```
Comment (ankan17) [inline src/components/animateNumber.vue]: Do not use !important everywhere. Only use it if it is required. And use consistent spacing. e.g.,
h3 {
    color: white;
}

**Comment by arc9693** [comment]
Changes have been made.

**Comment by ankan17** [comment]
1. Changes 3 and 4 are still not implemented.
2. The font size is not uniform. Even If it is intentional, it does not look good.
3. Remove the links from this section.
4. Make the text alignment justified.

**Pull request merged**